Assets/Shopify/UIToolkit/Themes/Generic/GenericSingleProductTheme.cs

```diff
@@ -66,9 +69,10 @@ public class GenericSingleProductTheme : MonoBehaviour, ISingleProductTheme {
         void ISingleProductTheme.OnShouldShowProduct(Product product, ProductVariant[] variants) {
             Title.GetComponent<Text>().text = product.title();
             Description.GetComponent<Text>().text = product.description();
```

Comment:
Did I write this this way? These should just be `Text` iirc but i might be a big dummy